articles/multifactor-authentication/api/_includes/_get_mfa_token.md

```diff
@@ -52,5 +55,3 @@ However, if MFA is required for this user, the `mfa_required` error will look li
   "mfa_token": "Fe26...Ha"
 }
 ```
-
-In the next step, use the MFA token (`mfa_token`) received in this response instead of the standard <dfn data-key="access-token">Access Token</dfn> to request association of a new authenticator.
```

Comment:
After the example it says `However, if MFA is required for this user, the mfa_required error will look like this:` which seems somewhat disconnected from the previous paragraph. Maybe it should say 'And the response will include the `mfa_required` error and a `mfa_token` that you can use when calling the `/mfa/associate` endpoint to enroll your first authenticator'.

Also, it would important to mention that the returned `mfa_token` can then be used to associate a new authenticator.